### PR TITLE
website/docs: release notes 2024.2: add note about RC

### PR DESCRIPTION
**Pull request by rissson**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docs/releases/2024/v2024.2.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ slug: /releases/2024.2
 
 :::::note
 2024.2 has not been released yet! We're publishing these release notes as a preview of what's to come, and for our awesome beta testers trying out release candidates.
+
```
Comment (kensternberg-authentik) [inline website/docs/releases/2024/v2024.2.md:8]: "To try the release candidate out" is, dunno, it sorta skips in my head.  Maybe "To try out the latest release candidate"?

**Review by kensternberg-authentik — APPROVED** [review]
A good addition. Language change suggested, but this can be approved in its current form.

File: website/docs/releases/2024/v2024.2.md (hunk 1 above)
Comment (rissson) [reply website/docs/releases/2024/v2024.2.md:8]: I agree yeah

File: website/docs/releases/2024/v2024.2.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ slug: /releases/2024.2
...
+To try the release candidate out, replace your Docker image tag with the latest release candidate number, such as 2024.2.0-rc1. You can find the latest one in [the latest releases on GitHub](https://github.com/goauthentik/authentik/releases). If you don't find any, it means we haven't released one yet.
```
Comment (rissson) [inline website/docs/releases/2024/v2024.2.md:9]:
```suggestion
To try out the release candidate, replace your Docker image tag with the latest release candidate number, such as 2024.2.0-rc1. You can find the latest one in [the latest releases on GitHub](https://github.com/goauthentik/authentik/releases). If you don't find any, it means we haven't released one yet.
```

**Pull request merged**